docs/spec/spec.md

```diff
@@ -29,37 +29,27 @@ This specification elaborates on the `Persist CLI Tool` commands.
 ## 2. Initializing the Bal Project with Persistence Layer
 
 ```bash
-bal persist init
+bal persist init --datastore="datastore" --module="module_name"
 ```
 
+ Following optional environment variables can be used with `bal persist init` command.
+ - datastore : used to indicate the preferred database client. Currently, only 'mysql' is supported.
+ - module : used to indicate the preferred name for the schema file. By default, this will be equal to the package name.
```

Comment:
Is this to indicate the schema file or to which module needs to be persistence enabled? 
@daneshk WDYT?

Reply:
No this doesn't have any connection with the schema file. This indicates the module in which the persistence is enabled and relevant codes are generated.

Reply:
updated in [1ecce62](https://github.com/ballerina-platform/persist-tools/pull/148/commits/1ecce626e9420d36be876cbb13c560ef8a376d09)